src/modules/uORB/SubscriptionInterval.hpp

```diff
@@ -118,15 +120,24 @@ class SubscriptionInterval
 
 	uint8_t		get_instance() const { return _subscription.get_instance(); }
 	orb_id_t	get_topic() const { return _subscription.get_topic(); }
-	uint32_t	get_interval() const { return _interval; }
 
-	void		set_interval(uint32_t interval) { _interval = interval; }
+	/**
+	 * Set the interval in microseconds
+	 * @param interval The interval in microseconds.
+	 */
+	void		set_interval_us(uint32_t interval) { _interval_us = interval; }
+
+	/**
+	 * Set the interval in milliseconds
+	 * @param interval The interval in milliseconds.
+	 */
+	void		set_interval_ms(uint32_t interval) { _interval_us = interval * 1000; }
```

Comment:
Why both? Can we settle on one?

Reply:
Trying to accommodate the existing usage. In logger and a few other modules setting orb intervals it's already using milliseconds. At the moment we really only need microseconds to accommodate the output modules (2500 us for 400 Hz pwm mixing and output).